gulpfile.js

```diff
@@ -10,6 +10,7 @@ var babel = require('gulp-babel')
 var eslint = require('gulp-eslint')
 var browserSync = require('browser-sync').create()
 var reload = browserSync.reload
+var config = require('./config.js')
```

Comment:
Snyggt, det borde finnas en config_example.js i repot, som man lägger in sin config och tar bort "_example"...annars är man lite i mörkret över vad som ska ligga i config-filen.

Reply:
Vi borde också uppdatera Readme.md och inkludera detta steg (att skapa config.js filen)